cmd/Makefile.am

```diff
@@ -98,13 +98,18 @@ endif
 
 
 if USING_PYTHON
-bin_SCRIPTS      += arc_summary     arcstat        dbufstat        zilstat
-CLEANFILES       += arc_summary     arcstat        dbufstat        zilstat
-dist_noinst_DATA += %D%/arc_summary %D%/arcstat.in %D%/dbufstat.in %D%/zilstat.in
+bin_SCRIPTS      += arc_summary     arcstat        dbufstat        zilstat \
+		     zfs_getnfs4facl zfs_setnfs4facl
+CLEANFILES       += arc_summary     arcstat        dbufstat        zilstat \
+		     zfs_getnfs4facl zfs_setnfs4facl
+dist_noinst_DATA += %D%/arc_summary %D%/arcstat.in %D%/dbufstat.in %D%/zilstat.in \
```

Comment:
Except for `zfs_getnfs4facl` and `zfs_setnfs4facl`, the other files (`arcstat`, `dbufstat`, etc) seem to be executable. For testing purposes, I explicitly granted execute permission to `zfs_getnfs4facl`, but I encountered the error: `ModuleNotFoundError: No module named 'libzfsacl'`.
Do I need to provide any additional flags during `./configure` to ensure libzfsacl is installed?

Reply:
It seems like on Debian based systems, `configure && make install` installs to site dir where python packages are installed and is not added to `sys.path` by default. I have updated to install to dist dir for Debian based systems, which should resolve the issue.